lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -168,10 +170,14 @@ export class CryptoOps implements ICrypto {
      * Removes cryptographic keypair from key store matching the keyId passed in
      * @param kid
      */
-    async removeTokenBindingKey(kid: string): Promise<boolean> {
+    async removeTokenBindingKey(kid: string): Promise<void> {
         await this.cache.removeItem(kid);
         const keyFound = await this.cache.containsKey(kid);
-        return !keyFound;
+        if (keyFound) {
```

Comment:
When will this be true? Only when `removeItem()` does not remove, are we expected to call this for all `removeItem` usages or is this extra handling for EAR for accuracy?